applications/StructuralMechanicsApplication/structural_mechanics_application.cpp

```diff
@@ -52,6 +52,9 @@ KratosStructuralMechanicsApplication::KratosStructuralMechanicsApplication():
     mShellThinCorotationalElement3D3N( 0, Element::GeometryType::Pointer( new Triangle3D3 <Node<3> >( Element::GeometryType::PointsArrayType( 3 ) ) ), true ),
     // Adding the membrane element
     mMembraneElement3D3N( 0, Element::GeometryType::Pointer( new Triangle3D3 <Node<3> >( Element::GeometryType::PointsArrayType( 3 ) ) ) ),
+	mMembraneElement3D4N( 0, Element::GeometryType::Pointer( new Quadrilateral3D4 <Node<3> >( Element::GeometryType::PointsArrayType( 4 ) ) ), false ),	
```

Comment:
Hi,
you can find the theory of the membrane element here:
https://www.dropbox.com/s/jya6kwvy77r9arl/tesi_globale%20da%20mandare%20.pdf?dl=0

or in 
http://onlinelibrary.wiley.com/doi/10.1002/nme.1266/abstract

having said this, adding a prestress to the element should be straightforward. In the long run however we should agree on a common way of doing it (see #402 )